v2/pkg/commands/build/base.go

```diff
@@ -165,8 +165,22 @@ func (b *BaseBuilder) CompileProject(options *Options) error {
 		}
 	}
 
+	commands := slicer.String()
+
+	compiler := options.Compiler
+	if options.Obfuscate {
+		if !shell.CommandExists("garble") {
+			options.Logger.Print("garble not found. Please install it with `go install mvdan.cc/garble@latest`. Using Go compiler instead.")
```

Comment:
If in dev mode, could you run that cmd on the host? 

"garble not found, Dev Mode installation, please wait.."
"Installing: `go install mvdan.cc/garble@latest`"

It will save you + developers time :)

Reply:
We have a policy of not executing installation code - makes people (rightly) nervous. Also can get tricky with permissions errors leading to a ton of tickets. We just give pointers :-)